sgkit/stats/hwe.py

```diff
@@ -140,6 +144,12 @@ def hardy_weinberg_test(
         where `N` is equal to the number of variants and the 3 columns contain
         heterozygous, homozygous reference, and homozygous alternate counts
         (in that order) across all samples for a variant.
+    call_genotype
+        Input variable name holding call_genotype.
+        As defined by :data:`sgkit.variables.call_genotype`.
```

Comment:
nit: This isn't a complete sentence which will be a little odd given how common "As defined by X." is.  I would suggest "Defined by X." instead.